pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -45,6 +50,10 @@ func checkHyperKitInstalled(networkMode network.Mode) func() error {
 
 func fixHyperKitInstallation(networkMode network.Mode) func() error {
 	return func() error {
+		if version.IsMacosInstallPathSet() {
+			return nil
+		}
+
```

Comment:
@guillaumerose Do we really need this check for this in the `fix` functions, if we have it on the `check`  then fix will not even run right?

Reply:
oups sorry didnt see it. 

Yes you're right. I really hope to remove this code in two/three releases.